PiwikTracker/Dispatcher.swift

```diff
@@ -4,7 +4,9 @@ public protocol Dispatcher {
     
     var baseURL: URL { get }
     
-    var userAgent: String? { get }
+    var userAgent: String? { get set }
+    
+    func setUserAgent(ua: String)
```

Comment:
@minitrue22 hmm, the thing is that dispatch will create default use agent by itself (SHOULD run in main thread). I think no need to create the default user agent out of the dispatch. What I want is that append some thing in the default user agent, maybe I should add an append method?